src/tenant/templates/tenant/email/verify_deck_request.txt

```diff
@@ -0,0 +1,11 @@
+Hello {{ first_name }},
+
+Thank you for requesting a new deck.
+
+Please verify your email within 1 hour by clicking this link:
```

Comment:
What happens if they don't verify, what can they do about it?  Can they request again?  I've had issues in the past where something like this times out and then I have to manually go in to the django shell and clean things up so they can try again. Please confirm.